src/Compilers/Core/Portable/CodeGen/ILBuilderEmit.cs

```diff
@@ -38,13 +38,13 @@ internal void EmitOpCode(ILOpCode code, int stackAdjustment)
 
         internal void EmitToken(string value)
         {
-            uint token = module == null ? 0xFFFF : module.GetFakeStringTokenForIL(value);
+            uint token = module?.GetFakeStringTokenForIL(value) ?? 0xFFFF;
             this.GetCurrentStream().WriteUint(token);
         }
 
         internal void EmitToken(Microsoft.Cci.IReference value, SyntaxNode syntaxNode, DiagnosticBag diagnostics)
         {
-            uint token = module == null ? 0xFFFF : module.GetFakeSymbolTokenForIL(value, syntaxNode, diagnostics);
+            uint token = module?.GetFakeSymbolTokenForIL(value, syntaxNode, diagnostics) ?? 0xFFFF;
```

Comment:
@jaredpar Might I suggest adopting :bulb: to indicate a non-blocking comment? I use [three specific emoji](https://github.com/openstacknetsdk/openstack.net/blob/master/CONTRIBUTING.md#emoji) to quickly provide context for the most common types of comment I leave.